cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -448,9 +448,8 @@ func (a *StaticAutoscaler) RunOnce(currentTime time.Time) caerrors.AutoscalerErr
 		return nil
 	}
 
-	if deletedNodes := a.deleteCreatedNodesWithErrors(); deletedNodes {
-		klog.V(0).Infof("Some nodes that failed to create were removed, skipping iteration")
-		return nil
+	if err = a.deleteCreatedNodesWithErrors(); err != nil {
+		klog.Errorf("Error while deleting created nodes with errors, skipping the iteration: %v", err)
```

Comment:
You're saying that you're skipping an iteration, but you're not actually doing it?